auth-api/src/auth_api/services/org.py

```diff
@@ -112,6 +114,11 @@ def create_org(org_info: dict, user_id,  # pylint: disable=too-many-locals, too-
             org.status_code = OrgStatus.PENDING_STAFF_REVIEW.value
             user = UserModel.find_by_jwt_token(token=token_info)
             Org.send_staff_review_account_reminder(user, org.id, origin_url)
+            # create a staff review task for this account
+            task_info = {'relationshipName': org.name,
```

Comment:
try to stick with consistent naming.. use `name` instead of relationshipName. Also I think here it should pass the task name [default to org name], relationship_type : Org, relationship_id : org id.